Mathlib/RingTheory/DedekindDomain/Basic.lean

```diff
@@ -9,14 +9,17 @@ import Mathlib.RingTheory.Polynomial.RationalRoot
 #align_import ring_theory.dedekind_domain.basic from "leanprover-community/mathlib"@"926daa81fd8acb2a04e15572c4ff20af2753c2ae"
 
 /-!
-# Dedekind domains
+# Dedekind rings and domains
 
-This file defines the notion of a Dedekind domain (or Dedekind ring),
-as a Noetherian integrally closed commutative ring of Krull dimension at most one.
+This file defines the notion of a Dedekind ring (domain),
+as a Noetherian integrally closed commutative ring (domain) of Krull dimension at most one.
 
 ## Main definitions
 
- - `IsDedekindDomain` defines a Dedekind domain as a commutative ring that is
+ - `IsDedekindRing` defines a Dedekind domain as a commutative ring that is
+   Noetherian, integrally closed in its field of fractions and has Krull dimension at most one.
+   `isDedekindRing_iff` shows that this does not depend on the choice of field of fractions.
+ - `IsDedekindDomain` defines a Dedekind domain as an integral domain that is
```

Comment:
I think you can golf this comment by simply saying that it is a Dedekind ring that is a domain.